packages/flutter/lib/src/widgets/navigation_toolbar.dart

```diff
@@ -83,15 +88,22 @@ const double _kMiddleMargin = 16.0;
 class _ToolbarLayout extends MultiChildLayoutDelegate {
   _ToolbarLayout({
     this.centerMiddle,
+    double requestedMiddleMargin,
     @required this.textDirection,
-  }) : assert(textDirection != null);
+  }) : assert(textDirection != null),
+       middleMargin = requestedMiddleMargin ?? _kMiddleMargin;
 
   // If false the middle widget should be start-justified within the space
   // between the leading and trailing widgets.
   // If true the middle widget is centered within the toolbar (not within the horizontal
   // space between the leading and trailing widgets).
   final bool centerMiddle;
 
+  /// The margin between leading and middle widgets.
```

Comment:
`[leading]`, `[middle]`

Reply:
These are not members of this class... Can I still link to them?

Reply:
Ah, no, nvm.